catalogs/sakip_sabanci_museum.yaml

```diff
@@ -3,28 +3,58 @@ metadata:
   data_path: sakip-sabanci
   schedule: "0 12 * * *"
 sources:
-  abidindino:
-    description: "ABİDİN DİNO ARCHIVE"
-    driver: oai_xml
-    args:
-      collection_url: http://cdm21044.contentdm.oclc.org/oai/oai.php
-      metadata_prefix: oai_dc
-      set: abidindino
+  abidin_dino:
+    description: "Abidin Dino Archive"
+    driver: iiif_json
     metadata:
-      data_path: sakip-sabanci/abidindino-xml
-      config: ssm_iiif_config
+      data_path: sakip_sabanci/abidin_dino
+      config: sakip_sabanci_museum
```

Comment:
I think this should be `ssm_abidindino`

Reply:
Same as bnf, I'm expecting only a single config for this collection so intentionally not adding collection names